generate-examples/issuer/.well-known/jwks.json

```diff
@@ -8,6 +8,29 @@
       "crv": "P-256",
       "x": "11XvRWy1I2S0EyJlyf_bWfw_TQ5CJJNLw78bHXNxcgw",
       "y": "eZXwxvO1hvCY0KucrPfKo7yAyMT6Ajc3N7OkAB6VYy8"
+    },
+    {
+      "kty": "EC",
+      "kid": "7rIUtD5K5Izrptr3ywgESTxhgJtJSeb06V42hg7WUDs",
```

Comment:
From the code in `generate-cert-chain.sh` I was expecting to see only one key added to this jwks -- can you help me understand where this one comes from?

Reply:
Weird. Is that the old encryption key you deleted? Maybe I didn't get the latest update... I'll double check

Reply:
Ah, that's a good bet -- and maybe you had a copy of it in your local git clone?

Reply:
Wasn't it, it was a stupid copy-paste error. Fixed by commit [b5dae6](https://github.com/smart-on-fhir/health-cards/pull/102/commits/b5dae62a92068340243f6d5404ece176f5d9fa0f).